front/app/containers/Admin/projects/projectFolders/components/ProjectFolderRow/FolderMoreActionsMenu.tsx

```diff
@@ -11,9 +11,14 @@ import { isNilOrError } from 'utils/helperUtils';
 export interface Props {
   folderId: string;
   setError: (error: string | null) => void;
+  setIsRunningAction?: (isLoading: boolean) => void;
```

Comment:
```suggestion
  setIsRunningAction: (isLoading: boolean) => void;
```
We don't have instances of FolderMoreActionsMenu where we don't need to add this, so let's make it stricter for now?

Reply:
That is actually intentional so this component and the other one aren't coupled soo much with their usage. Having it like this decouples this use case as it is an outlier. Normally, this loading state business should be with in this component